### PR TITLE
Update copyright for 2020

### PR DESCRIPTION
**Pull request by ryanRfox**
Update copyright to cover 2019

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Copyright (c) 2015-2016 Cryptonomex Inc. <contact@cryptonomex.com>
-Copyright (c) 2015-2017 contributors <CONTRIBUTORS.txt>
```
Comment (oxarbitrage) [inline LICENSE.txt]: - We have 2 lines of Copyright ?
- We have a reference to CONTRIBUTORS.txt witch is empty.

I think we can do something more like: https://github.com/bitshares/bitshares-ui/blob/develop/LICENSE.md for the header.

**Comment by xeroc** [comment]
python-bitshares uses an [AUTHORS](https://github.com/bitshares/python-bitshares/blob/master/AUTHORS) file that is automatically created by this [Makefile entry](https://github.com/bitshares/python-bitshares/blob/master/Makefile#L53)

Could do the same with contributors

**Comment by sschiessl-bcp** [comment]
I like that script from Fabian. Nevertheless, in the UI repository we have 
"Copyright (c) 2019 The BitShares UI worker and and contributors."

**Comment by ryanRfox** [comment]
I too like that makefile script > AUTHORS provided by @xeroc . However, I don't see the linkage between that file and the LICENSE file. I was expecting the Copyright therein to reference the AUTHORS file. Referencing the Core WP feels to vague for a copyright. I remain open to identifying what the best solution may be.

**Comment by xeroc** [comment]
Worth a read:
* https://opensource.stackexchange.com/questions/5778/why-do-licenses-such-as-the-mit-license-specify-a-single-year
* https://softwareengineering.stackexchange.com/questions/157968/how-to-manage-a-copyright-notice-in-an-open-source-project

There's a difference between licensing code to you and assigning copyright to you, and in this case the code is licensed to you while the contributor still holds the copyright.

Since their contributions are under the MIT license, though, you can later apply a different license to the combined work, as long as you preserve their copyright notice and the MIT license.

Hence, for a project that has a central LICENSE file but many contributors, it makes sense to have the copyright notice declare that many contributors are owners of the code.

Examples:
* `Copyright Node.js contributors. All rights reserved.`
* `Copyright (C) 2012-2018 by various contributors (see AUTHORS)`
* `Copyright (c) 2007 - 2018, Daniel Stenberg with many contributors, see AUTHORS
file.`
* `Copyright Joyent, Inc. and other Node contributors.`
* `Copyright (c) 2015-present libuv project contributors.`
* `Copyright (c) 1998-2018 The OpenSSL Project. All rights reserved.`

It may make sense to cleanup the code and remove the license boilerplate in favor of having a central spot for the license anyways.

**Comment by ryanRfox** [comment]
Thanks @xeroc for the references. I'll read up and determine more holistically how we will maintain our attributions. Moving this to the next Feature Release.

**Comment by pmconrad** [comment]
* Rebased on latest develop
* added CONTRIBUTORS.txt as suggested by @xeroc 
* added .mailmap file for merging commits from the same person with different name/email
* assigned copyright to BBF

Question: what about license headers in individual source files?

**Pull request merged**